source/sdk/math.ooc

```diff
@@ -358,6 +318,9 @@ extend Float {
 }
 
 extend LDouble {
+	pi ::= static 3.14159_26535_89793_23846_26433_83279 as This
+	e ::= static 2.718281828459045235360287471352662497757247093699959574966 as This
```

Comment:
Adding L suffix to float literal works in C, to specify a long double.
I'm not aware of any suffix for unsigned short in standard C. There is a suffix for microsoft compilers though, but I don't remember what was it, maybe `10u` or something like that.